src/dmd/lexer.d

```diff
@@ -1134,11 +1134,16 @@ class Lexer
                     p++;
                     Token n;
                     scan(&n);
+                    if (n.value == TOK.int32Literal)
```

Comment:
Restrict to just compiling C?

Reply:
I thought about that, but if some nutburger was preprocessing code before compiling it with D, well, whatever! Maybe someone will find an interesting use case.

Reply:
If it's not restricted to `ImportC`, then it changes the D specification. And it needs to come with a spec PR. And it goes against our contributing guidelines that PR should not conflate bug fixes / enhancements together.

Reply:
Fair 'nuf, though these are both enhancements.